cfgov/unprocessed/apps/retirement/js/views/graph-view.js

```diff
@@ -1,5 +1,6 @@
-import fetch from '../wizards/fetch-api-data.js';
+import fetchApiData from '../wizards/fetch-api-data.js';
 import getModelValues from '../wizards/get-model-values.js';
+import update from '../wizards/update-model.js';
```

Comment:
Could be named `updateModel` to match naming of the other imports. `update` is also generic.

Reply:
I have a large-ish thing brewing for this, will actually open a new PR ^__^